### PR TITLE
Add links "How javascript works"

### PR DESCRIPTION
**Pull request by espozbob**
Every javascript programmer must know and care about it, no matter low level or high level.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,8 @@ But knowing the stuff will help you become better! :muscle:*
 - :book: [Practical Object Oriented Design in Ruby](http://www.poodr.com/)
 - :movie_camera: [Evolutionary Software Architectures](https://www.youtube.com/watch?v=CglSFhwbI3s)
 - [System Design: A Primer](https://github.com/donnemartin/system-design-primer)
+- :page_facing_up: [How JavaScript works: part-1](https://blog.sessionstack.com/how-does-javascript-actually-work-part-1-b0bacc073cf), [2](https://blog.sessionstack.com/how-javascript-works-inside-the-v8-engine-5-tips-on-how-to-write-optimized-code-ac089e62b12e), [3](https://blog.sessionstack.com/how-javascript-works-memory-management-how-to-handle-4-common-memory-leaks-3f28b94cfbec), [4](https://blog.sessionstack.com/how-javascript-works-event-loop-and-the-rise-of-async-programming-5-ways-to-better-coding-with-2f077c4438b5)
+- :movie_camera: [How JavaScript works(video version)](https://www.youtube.com/watch?v=8aGhZQkoFbQ)
```
Comment (eamanu) [inline README.md]: One item per Pull Request.

See the CONTRIBUTING.md file

**Comment by espozbob** [comment]
@eamanu Fixed!

File: README.md [hunk 2]
```diff
@@ -92,6 +92,7 @@ But knowing the stuff will help you become better! :muscle:*
 - :book: [Practical Object Oriented Design in Ruby](http://www.poodr.com/)
 - :movie_camera: [Evolutionary Software Architectures](https://www.youtube.com/watch?v=CglSFhwbI3s)
 - [System Design: A Primer](https://github.com/donnemartin/system-design-primer)
+- :page_facing_up: [How JavaScript works: part-1](https://blog.sessionstack.com/how-does-javascript-actually-work-part-1-b0bacc073cf), [2](https://blog.sessionstack.com/how-javascript-works-inside-the-v8-engine-5-tips-on-how-to-write-optimized-code-ac089e62b12e), [3](https://blog.sessionstack.com/how-javascript-works-memory-management-how-to-handle-4-common-memory-leaks-3f28b94cfbec), [4](https://blog.sessionstack.com/how-javascript-works-event-loop-and-the-rise-of-async-programming-5-ways-to-better-coding-with-2f077c4438b5)
```
Comment (eamanu) [inline README.md]: Great

**Comment by mr-mig** [comment]
This one may be very controversial, but JavaScript has actually hit mainstream and is seemingly everywhere (and is "a new Java" in sense). 
Merging

**Comment by mr-mig** [comment]
Thanks for contributing @espozbob and sorry that it took so much time to merge

**Pull request merged**